docs/reference/cluster_manifest.md

```diff
@@ -359,3 +363,28 @@ CPU and memory limits for the sidecar container.
 * **memory**
   memory limits for the sidecar container. Optional, overrides the
   `default_memory_limits` operator configuration parameter. Optional.
+
+## Custom TLS certificates
+
+By default, the spilo image generates its own TLS certificate during startup.
+This certificate is not secure since it cannot be verified and thus doesn't
+protect from active MITM attacks.
```

Comment:
I would add: "A custom certificate can be specified under the `tls` top-level key."

Then it's aligned to the other groups. I would also remove the first mention of the `tls` field under top-level parameters. We also don't do it for other groups such as `patroni` etc.